bin/main/io/seqera/wave/service/metric/model/MetricsResponse.groovy

```diff
@@ -22,12 +22,12 @@ package io.seqera.wave.service.metric.model
  *
  * @author Munish Chouhan <munish.chouhan@seqera.io>
  */
-class GetOrgCountResponse {
+class MetricsResponse {
```

Comment:
It's a good pattern keep request and response object following the pattern: `<VERB><ENTITITY>Request/Response`.

Reply:
Let me know when it's solved